### PR TITLE
XIVY-10481 product idea as template

### PR DESCRIPTION
**Pull request by ivy-rew**
as discusses on last scrum-of-scrum meeting:
an new issue template, to describe ideas on future market-products and their capabilities.
existing-issue-template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/product-idea.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+about: Share a new market-produkt idea that would benefit you.
+title: [name] product
+labels: ''
+assignees:
+  - ivy-rew
+  - alexsuter
+  - ivy-lmu
+  - weissreto
+  - ivy-bb
+  - axonivy-market
```
Comment (alexsuter) [inline .github/ISSUE_TEMPLATE/product-idea.md:12]: axonivy-market?

Comment (alexsuter) [reply .github/ISSUE_TEMPLATE/product-idea.md:12]: maybe only one or two guys should be the gatekeeper?

**Review by alexsuter — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/product-idea.md (hunk 1 above)
Comment (ivy-rew) [reply .github/ISSUE_TEMPLATE/product-idea.md]: remove axonivy-market. 

And yes, we should define the responsibilities also internally. Now we have a growing eco-system and nobody is really responsible. Despite personal efforts :)

**Comment by ivy-rew** [comment]
I'll merge it as is; as it's easier to try it out by yourself then ... however, feel free to share your feedback or fire adapting PRs.

**Review by weissreto — COMMENTED** [review]
👍 Have to migrate some market JIRA issues to this.

**Pull request merged**